app/controllers/application_controller.rb

```diff
@@ -14,19 +14,11 @@ class ApplicationController < ActionController::Base
 
   helper_method :blackcat_config
 
-  before_action :flash_readonly, :flash_alert
-
-  def blackcat_config
-    @blackcat_config ||= BlackcatConfig::Builder.new
-  end
+  before_action :flash_alert
 
   private
 
-    def flash_readonly
```

Comment:
@cdmo how does the readonly flash will be displayed?

Reply:
is the idea set the `readonly` to true and also set the `alert`? that would work I think, and liked this idea

Reply:
But I could not get it to work like that 🤔

Reply:
yup, basically just make one channel for setting that. However, I have a new thought based on #559. I think we can just use the announcement bar. Like let's just replace that when there is an alert. Obviously setting flash :error seems to have unintended side affects.

Reply:
gone now due to #559